mypy.ini

```diff
@@ -0,0 +1,15 @@
+[mypy]
+warn_unused_configs = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+warn_no_return = True
+warn_unreachable = True
+strict_equality = True
+
+show_error_context = True
+show_column_numbers = True
+show_error_codes = True
+pretty = True
+
+[mypy-pygraphviz.*]
+ignore_missing_imports = True
```

Comment:
I'd rather have this in the setup.cfg too instead of an additional file

Reply:
moved to toml file